spec/unit/lib/cloud_controller/diego/app_recipe_builder_spec.rb

```diff
@@ -570,6 +570,94 @@ module Diego
             end
           end
 
+          context 'readiness health check' do
+            context 'when the readiness health check type is not set' do
+              before do
+                process.readiness_health_check_type = ''
```

Comment:
Is this case possible?  I thought the migration that adds a default to the column would mean that all processes would initially have a `readiness_health_check_type` of process

`add_column :readiness_health_check_type, String, default: 'process'`

https://github.com/cloudfoundry/cloud_controller_ng/pull/3361/files#diff-bc587143ec711deb3455361c0dfd4e832be04b77dd46edc603f731e53253481aR6

Reply:
I don't _think_ so. There is logic in the "normal" health check, where an empty health check type defaults to "port" (I don't even know if this is possible anymore? Perhaps this is part of the remnants of when "none" was allowed?). So I did the same, but for readiness health check types and defaulting to process. I will remove it.

Reply:
You are right. This test should be about when the readiness health check defaults to process. Good catch. Fixed.